analysis/benchmark_results.py

```diff
@@ -45,6 +48,19 @@ def _prefix_with_benchmark(self, filename):
     def _get_full_path(self, filename):
         return os.path.join(self._output_directory, filename)
 
+    def _get_experiment_filestore_path(self, fuzzer_name):
+        return coverage_data_utils.get_fuzzer_filestore_path(
+            self._benchmark_df, fuzzer_name)
+
+    def get_filestore_name(self, fuzzer_name):
+        """Returns the filestore name of the |fuzzer_name|."""
+        filestore_path = self._get_experiment_filestore_path(fuzzer_name)
+        gcs_prefix = 'gs://'
+        gcs_http_prefix = 'https://storage.googleapis.com/'
```

Comment:
:-( we don't make gsutil a dependency for fuzzbench. This method of downloading the data over http allows us not to do this but doesn't support a unix filesystem as a filestore. @lszekeres thoughts?

Reply:
Actually I think my understanding of what this is used for is wrong.
@Shadoom7 is this function only used for displaying a link in a report?
Maybe we can implement some of this as a helper function filestore utils that just assumes the http link and we can refactor later when we actually support local experiments.

Reply:
Yes, it's only used to display the link. Do you mean we should add the function in `filestore_utils` to return the http link we just call that function here, ignoring the case where filestore_path is local?

Reply:
Yes exactly.

Reply:
Should we put it into `filestore_utils` though? I thought `filestore_utils` only contains possible terminal commands that are related to filestore.

Reply:
I think Jonathan's original question is more relevant to line 63 of `coverage_data_utils.py` (`filestore_utils.cp(src_file, dst_file)`), which copies the JSON file first with gsutil cp or with cp. If we want to eliminate `generate_report.py`'s dependency on `gstuil` (which would be a good idea), what would could do is:
1) in case of gs:// path, get the file via http from storage.googleapis.com (eg with urllib) 
2) in case of a local path, simply open the file directly

wdyt?

Reply:
> I think Jonathan's original question is more relevant to line 63 of `coverage_data_utils.py` (`filestore_utils.cp(src_file, dst_file)`), which copies the JSON file first with gsutil cp or with cp. If we want to eliminate `generate_report.py`'s dependency on `gstuil` (which would be a good idea), what we could do is:
> 
> 1. in case of gs:// path, get the file via http from storage.googleapis.com (eg with urllib)
> 2. in case of a local path, simply open the file directly
> 
> wdyt?

Yeah can come later. I sort of wished we investigated alternatives to gsutil more since this is another edge case we have to deal with on our own.